### PR TITLE
BETA: Register lz.is-a.dev

### PR DESCRIPTION
**Pull request by littletheworld**
Added `lz.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by littletheworld** [comment]
why my PR is failed 🥲🥲

File: domains/ilz.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "littletheworld",
+           "email": "littlehaofficial@gmail.com",
+           "discord": "810846876592242719"
+        },
+    
+        "record": {
+            "A": "104.21.85.194"
```
Comment (CuteDog5695) [inline domains/ilz.json:9]: IP needs to be in an array

File: domains/lnk.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
         },
     
         "record": {
-            "CNAME": "cname.short.io"
+            "A": "104.21.85.194"
```
Comment (CuteDog5695) [inline domains/lnk.json:9]: IP needs to be in an array

File: domains/lnk.json
Comment (CuteDog5695) [inline domains/lnk.json:1]: lnk.is-a.dev already exists so you cant get this domain or change it (since it belongs to a different user)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/lnk.json
Comment (CuteDog5695) [reply domains/lnk.json:1]: actually nvm you are the user, my bad

**Pull request closed without merging**